### ISSUE TITLE
E0092 needs to be updated to new format

### ISSUE THREAD

**Issue by jonathandturner** [rust-lang/rust#35229]
From: src/test/compile-fail/E0092.rs

Error E0092 needs a span_label, updating it from:

```
error[E0092]: unrecognized atomic operation function: `foo`
  --> src/test/compile-fail/E0092.rs:13:5
   |
13 |     fn atomic_foo(); //~ ERROR E0092
   |     ^^^^^^^^^^^^^^^^
```

To:

```
error[E0092]: unrecognized atomic operation function: `foo`
  --> src/test/compile-fail/E0092.rs:13:5
   |
13 |     fn atomic_foo(); //~ ERROR E0092
   |     ^^^^^^^^^^^^^^^^ unrecognized atomic operation
```

**Comment by Vassah** [comment]
I'll be tackling this one over the next few days.

**Comment by Vassah** [comment]
Addressed in [#35528](@jonathandturner) pending merge.
